.github/workflows/ci.yml

```diff
@@ -102,7 +102,8 @@ jobs:
           import os
           ref = os.environ['GITHUB_REF']
           prod = os.environ['PRODUCTION_BRANCH']
-          if ref == prod:
+          staging = os.environ['STAGING_BRANCH']
+          if ref == prod or ref == staging:
```

Comment:
@seaerchin is this a temp change or perm? Also what does "efs has issues mounting" mean, is it an unexpected error? could you provide a little more context?

Reply:
This is temp for now, will try to debug efs mounting when I'm back 

The problem is that ecs itself resides in a different vpc than efs. I've created a peering connection from the eb private vpc to the compute vpc (together with a ingress rule that allows NFS traffic from ecs subnet to eb private subnet). Using aws reachability analyzed shows that nfs traffic over the peering connection reaches our efs.

However, at deployment time, what happens is that the filesystem id of our efs is unable to be resolved. We already created a mount target for our efs inside the existing private subnet, so we can't create another one in the same AZ. 

I'm not entirely sure what's happening here (deployment logs on ecs just shows that the mount failed), maybe we just need to allow egress traffic but (when I'm back), gonna try a few things and if fail, just create our ecs inside the existing vpc...